roles/download/defaults/main.yml

```diff
@@ -23,16 +23,14 @@ etcd_version: v3.0.6
 # after migration to container download
 calico_version: "v1.0.0"
 calico_cni_version: "v1.5.5"
-weave_version: v1.6.1
```

Comment:
Their tagged releases do not include `v` https://hub.docker.com/r/weaveworks/weave-kube/tags/